src/main/java/com/iexec/common/utils/FileHelper.java

```diff
@@ -420,11 +420,7 @@ private static void printFile(File file, int indent, StringBuilder sb) {
     }
 
     private static String getIndentString(int indent) {
-        StringBuilder sb = new StringBuilder();
-        for (int i = 0; i < indent; i++) {
-            sb.append("│   ");
-        }
-        return sb.toString();
+        return "│   ".repeat(indent);
```

Comment:
Nice

Reply:
Yeah, I didn't know it was a thing in Java 11 but that looks way better than the ugly loop :)